### PR TITLE
New Smoke Particles!

### PR DESCRIPTION
**Pull request by techno-sam**

### CODE REVIEW

File: common/src/main/java/com/railwayteam/railways/config/CClient.java [hunk 1]
```diff
@@ -16,13 +18,21 @@ public class CClient extends ConfigBase {
     public final ConfigFloat trackOverlayOffset = f(0.0f, -256.0f, 256.0f, "trackOverlayOffset", Comments.trackOverlayOffset);
     public final ConfigBool useDevCape = b(true, "useDevCape", Comments.useDevCape, Comments.useDevCape2);
 
+
     // smoke
     public final ConfigGroup smoke = group(1, "smoke", Comments.smoke);
+    public final ConfigEnum<SmokeType> smokeType = e(SmokeType.OLD, "smokeType", Comments.smokeType);
```
Comment (IThundxr) [inline common/src/main/java/com/railwayteam/railways/config/CClient.java]: Shouldnt default smoke type be the new one?

**Review by IThundxr — COMMENTED** [review]
Looks good to me apart from a small little thing

File: common/src/main/java/com/railwayteam/railways/config/CClient.java (hunk 1 above)
Comment (techno-sam) [reply common/src/main/java/com/railwayteam/railways/config/CClient.java]: oops. yes, it should. I thought I set it to be

**Review by IThundxr — APPROVED** [review]
Looks good to me!

**Pull request merged**